src/main/java/phase1/runnable/Runnable.java

```diff
@@ -19,6 +19,7 @@ public static void main(String[] args) {
         bm.countMonthly("05");      // 월별 입출금 건수
                                     // 지출이 가장 높은 상위 10건
         bm.itemExpense();           // 많이 소비한 항목
+        // 데모 데이터를 생성해야 합니다.
     }
 
 
```

Comment:
24,25번 라인은 없어져야 할 것 같네요. 삭제부탁드립니다.
```suggestion
// 삭제
// 삭제
```

Reply:
바로 삭제하겠습니다.